addon/adapters/json-api.js

```diff
@@ -99,7 +100,7 @@ var JSONAPIAdapter = RESTAdapter.extend({
     @return {Promise} promise
   */
   findMany(store, type, ids, snapshots) {
-    if (isEnabled('ds-improved-ajax')) {
+    if (isEnabled('ds-improved-ajax') && !this._hasCustomizedAjax()) {
```

Comment:
This only works if we support https://github.com/ember-cli/babel-plugin-feature-flags/pull/4. Are the necessary dependencies in `ember-data` updated so this is satisfied? I'm just asking; haven't had time to follow recent developments in `ember-data` too closely...